recipes/folly/all/patches/0017-std-hash.patch

```diff
@@ -0,0 +1,180 @@
+diff --git a/folly/Uri-inl.h b/folly/Uri-inl.h
```

Comment:
```suggestion
https://github.com/facebook/folly/pull/1587
diff --git a/folly/Uri-inl.h b/folly/Uri-inl.h
```

This is how you can add the link to the patch itself.

Reply:
Oh, nice, thanks.